src/edit/WebGlDistort.js

```diff
@@ -0,0 +1,182 @@
+// http://stackoverflow.com/questions/4998908/convert-data-uri-to-file-then-append-to-formdata

```

Comment:
Whoa! Are these files copied in from webgl-distort? I think if we include the built source from that library via npm `require`, we needn't actually copy in or reproduce the source code within leaflet DistortableImage - you know? I think it may be as simple as the example on https://github.com/jywarren/webgl-distort

What do you think?

Reply:
And thanks again for your help!!!

Reply:
Yeah, I switched over to just including them - I think I'm getting closer, just need to set up some testing.